cmd/bucket-notification-utils.go

```diff
@@ -259,15 +259,14 @@ func validateNotificationConfig(nConfig notificationConfig) APIErrorCode {
 // - kafka
 // - webhook
 func unmarshalSqsARN(queueARN string) (mSqs arnSQS) {
-	mSqs = arnSQS{}
 	strs := strings.SplitN(queueARN, ":", -1)
 	if len(strs) != 6 {
-		return mSqs
+		return
```

Comment:
Even though it is supported, we do not use this style ie we use `return mSqs` style everywhere in minio code base.  Is it changed now?